### ISSUE TITLE
Npc undo

### ISSUE THREAD

**Issue by ilikestohack** [CitizensDev/Citizens2#2153]
Hello I am running a server version 1.15.2 paper and the latest citizens github release and I was wondering if we could get a /npc undo for example if you make a powertool and copy a bunch of npcs to have one cmd to undo without selecting and despawning 100 npcs

**Comment by fullwall** [comment]
As in, to remove the last created NPCs in order?

**Comment by ilikestohack** [comment]
Yes to remove last copied or created npcs or if devs want to not just
creations but changes too

**Comment by CDWimmer** [comment]
I accidentally just `/npc remove all` instead of `/npc command remove` and have no way to undo this so I'd bloody love an undo command right now 🙃

**Comment by mcmonkey4eva** [comment]
@CDWimmer Well, there's your lesson to always have backups of all your server files! Ideally making new full backups regularly (once a week for example), that way if everything goes horribly wrong you'll only lose things you did in the past day or two and nothing more.

**Comment by CDWimmer** [comment]
Indeed, fortunately I only had a couple scattered in an admin room as I hadn't got around to using them yet but perhaps that command should have an "are you sure" step to it
